### PR TITLE
 Fixed graphd crash

### PR DESCRIPTION
**Pull request by monadbobo**
Fixed crash caused when not specifying edge type (over *) and setting reversely.

### CODE REVIEW

File: src/graph/test/GoTest.cpp [hunk 1]
```diff
@@ -917,6 +917,25 @@ TEST_F(GoTest, ReverselyOneStep) {
         };
         ASSERT_TRUE(verifyResult(resp, expected));
     }
+    {
+        cpp2::ExecutionResponse resp;
+        auto query = "GO FROM hash('Tim Duncan') OVER * REVERSELY "
+                     "YIELD like._src";
```
Comment (CPWstatic) [inline src/graph/test/GoTest.cpp]: Would you please add an UT with multi-edges?  I am not quite sure if this test case could cover it.

File: src/graph/test/GoTest.cpp [hunk 2]
```diff
@@ -380,7 +380,90 @@ TEST_F(GoTest, MULTI_EDGES) {
         };
         ASSERT_TRUE(verifyResult(resp, expected));
     }
-
+    {
+        cpp2::ExecutionResponse resp;
```
Comment (laura-ding) [inline src/graph/test/GoTest.cpp]: Could you add a test to cover reversely multi-edge and the result has different props on the different edges?

File: src/graph/test/GoTest.cpp [hunk 3]
```diff
@@ -380,7 +380,117 @@ TEST_F(GoTest, MULTI_EDGES) {
...
+        ASSERT_TRUE(verifyResult(resp, expected));
+    }
+    {
+        cpp2::ExecutionResponse resp;
+        auto *fmt = "GO FROM %ld OVER * REVERSELY yield like.likeness, teammate.start_year, "
+                    "serve.start_year, $$.player.name";
+        auto &player = players_["Manu Ginobili"];
+        auto query = folly::stringPrintf(fmt, player.vid());
+        auto code = client_->execute(query, resp);
+        ASSERT_NE(cpp2::ErrorCode::SUCCEEDED, code);
```
Comment (CPWstatic) [inline src/graph/test/GoTest.cpp]: Why are we not check the result here? We could also yield source vertex props.

**Comment by jude-zhu** [comment]
close #1364
